ch07.adoc

```diff
@@ -15,7 +15,12 @@ For cases when data pertain to geospatial features with highly variable geometry
 To represent cells we add the attribute **`bounds`** to the appropriate coordinate variable(s).
 The value of **`bounds`** is the name of the variable that contains the vertices of the cell boundaries.
 We refer to this type of variable as a "boundary variable."
-__A boundary variable will have one more dimension than its associated coordinate or auxiliary coordinate variable.__ The additional dimension should be the most rapidly varying one, and its size is the maximum number of cell vertices.
+__A boundary variable must have one more dimension than its associated coordinate or auxiliary coordinate variable.__ 
+We refer to the additional dimension as the "vertex dimension".
+The vertex dimension must be the most rapidly varying one, and its size is the maximum number of cell vertices.
```

Comment:
perhaps:

"...The vertex dimension must be the last (most rapidly varying) one, ..."

Reply:
Thanks, @ChrisBarker-NOAA. Update in #163.